libs/wingsdk/src/platform/platform-manager.ts

```diff
@@ -232,9 +232,11 @@ export function _loadCustomPlatform(customPlatformPath: string): any {
     script.runInContext(context);
     return new (platformModule.exports as any).Platform();
   } catch (error) {
+    const hint = isScoped
+      ? `Ensure you have installed the platform package by running 'npm install ${customPlatformPath}'`
+      : "Ensure the path to the platform is correct";
     console.error(
-      "An error occurred while loading the custom platform:",
-      error
+      `An error occurred while loading the custom platform: ${customPlatformPath}\n\n(hint: ${hint})`
```

Comment:
Please add instructions or doclink and suggest that the platform provider/extension might not be installed...

Reply:
@eladb Just to be clear, when you suggest adding instructions are you referring to how to install platform extension? Currently in this PR if a platform is not installed, lets say `@winglang/platform-aws-cdk` the error message looks like this:
```bash
mywing compile -t @winglang/platform-awscdk
An error occurred while loading the custom platform: @winglang/platform-awscdk

(hint: Ensure you have installed the platform provider by running 'npm install @winglang/platform-awscdk')
```